frontend/apps/cloudstorage/dropbox.lua

```diff
@@ -111,8 +111,8 @@ function DropBox:config(item, callback)
         height = Screen:getHeight() * 0.2,
         input_type = "text",
     }
-    self.settings_dialog:onShowKeyboard()
     UIManager:show(self.settings_dialog)
+    self.settings_dialog:onShowKeyboard()
```

Comment:
Instead of changing the order, have you tried making it a modal widget instead with `modal = true,`? If that works it'd be simpler and less error prone.

Reply:
We can leave the order changed either way. It does seem slightly illogical to show the keyboard first. ;-) In any case it seems like a good thing to test. :-)

Reply:
Oh that's awesome, exactly what is needed for the virtualKeyboard.

I already switched all the line, so I kinda want to keep it, I will look into it.